src/events/VoiceState.ts

```
@@ -15,7 +16,24 @@ export class Voice extends EventHandler<Events.VoiceStateUpdate> {
         const member = newState.member
         const category = settings?.voiceCategory
         if (!settings || !member || !category) return
+        const oldId = oldState.channelId
+        if (oldId && (await database.findChannel(oldId))?.id) {
```

ESLint

[error] Do not access a member directly from an await expression.

(unicorn/no-await-expression-member)